Wizard Academy UX Fix

Kyep opened the pull request:
Entering the Gateway to the Wizard Academy mission spawns you at one of a number of random points.
Two of these points are in this diagram, marked with blue Xes inside red circles:


![swf_academy_spawnpoints](https://user-images.githubusercontent.com/16434066/45143289-61502880-b1aa-11e8-9609-c4ed3e7ae822.PNG)

The north one by the two welding tanks is a terrible spawn point, because it limits your options to:
- welding the first tank, and dying to the blast, fastmos blender, and/or the three mimics disguised as crates in the room who will swarm you
- remaining stuck there until someone rescues you, which isn't guaranteed to happen ever, and even if someone finds you, and tries, they still have to fight three mimics
- using a full toolset to cut through the wall to your left, then a space suit to find another entry point, whilst hoping that you don't die to the named megacarp located within eyesight of the airlock. Not realistic.

The south one is essentially 'spawn in a class cage right next to a bunch of hostile space bears, proceed to get immediately mauled and die'.

This PR moves the first spawn point to the next room (the pink X) and simply deletes the second.

No longer will admins routinely get "I entered gateway and it spawned me in an impossible place, help, I'm stuck!" ahelps/prayers. Nor will players walk in and almost immediately die with nothing they could do about it.

🆑 Kyep
fix: Fixed bad spawn points in the Wizard Academy away mission.
/🆑

taukausanake commented:
For the top one, isn't tearing down the mech charger and _then_ getting murderized by mimics also an option? You make it sound more impossible than it seems.

That seems like a good new location. Maybe make it so they all spawn in a different chair

Kyep commented:
In all the times I've seen people land in this spot, I have never, not even once, seen someone tear down a charger and make it out that way. Even if it is possible, which I rather doubt, it is so prohibitively difficult that it may as well not exist. It isn't going to stop the "help, I'm stuck" ahelps/prayers from people who land in this spot, nor is it giving them a fair shake.

Basic game design is that players should generally be able to run away from situations they cannot overcome. To randomly put them in an impossible situation where they'll die and then have to sit out the next hour and a half is simply bad design. 

If you look at where I moved that spawn point, I moved it to simply the next room. Players can still enter that room and die to the mimics. Its still very possible for them to lose. Probable even. The difference is that after this fix, the player experience is now fair - they live or die by their own actions, rather than the RNG of where they warp in at.
